include/manif/impl/lie_group_base.h

```diff
@@ -601,10 +602,13 @@ LieGroupBase<_Derived>::adj() const
 }
 
 template <typename _Derived>
+template <typename _EigenDerived>
 typename LieGroupBase<_Derived>::Vector
-LieGroupBase<_Derived>::act(const Vector& v,
-                            OptJacobianRef J_vout_m,
-                            OptJacobianRef J_vout_v) const
+LieGroupBase<_Derived>::act(
+  const Eigen::MatrixBase<_EigenDerived>& v,
+  tl::optional<Eigen::Ref<Eigen::Matrix<Scalar, Dim, DoF>>> J_vout_m,
+  tl::optional<Eigen::Ref<Eigen::Matrix<Scalar, Dim, Dim>>> J_vout_v
+) const
 {
   return derived().act(v, J_vout_m, J_vout_v);
```

Comment:
what about the signature in derived classes?

Reply:
There are correct and unit tested. The issue here really is when calling `act` from a `LieGroup<Derived>` object. This use-case is not unit tested at the moment but I'll look into that.